meilisearch/client.py

```diff
@@ -464,3 +469,67 @@ def wait_for_task(
             An error containing details about why Meilisearch can't process your request. Meilisearch error codes are described here: https://docs.meilisearch.com/errors/#meilisearch-errors
         """
         return wait_for_task(self.config, uid, timeout_in_ms, interval_in_ms)
+
+    def generate_tenant_token(
```

Comment:
I saw the discussion about making expires_at and api_key named only arguments. You can do that in Python. 

```py
def generate_tenant_token(
    self,
    search_rules: Union[Dict[str, Any], List[str]],
    *,
    expires_at: Optional[int] = None,
    api_key: Optional[str] = None,
) –> str:
    …
```

Everything after the * is named only.